launch_param_builder/launch_param_builder.py

```diff
@@ -71,5 +71,12 @@ def parameter(self, parameter_name: str, parameter_value: ParameterValueType):
         self._parameters[parameter_name] = parameter_value
         return self
 
+    def path_parameter(self, parameter_name: str, file_path: str):
+        self._parameters[parameter_name] = str(self._package_path / file_path)
+        return self
+
+    def path_parameter_raw(self, file_path: str):
+        return str(self._package_path / file_path)
+
```

Comment:
What's the use case for this? I don't see it adding any parameter

Reply:
For purposes when the path needs to be propagated as argument (e.g. .rviz files for rviz configuration or .world file for ignition). It is not a ros parameter, but is quite often needed in launch file system